### PR TITLE
dHtml english

### PR DESCRIPTION
**Pull request by kathyoleary**
:green_apple: 

ready to go

### CODE REVIEW

**Comment by ericanaughton** [comment]
good job. just need to remove the comments and  remove the audio placeholder.

File: elements/html.html [hunk 1]
```diff
@@ -0,0 +1,59 @@
+<article class="article">
+  <header class="article-header">
+    <h2>
+      <code>
+        <!-- Actual Tag Name; Whatever the tag name is.  -->
```
Comment (jremsik-gpsw) [inline elements/html.html]: These comments should be removed.

File: elements/html.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        &lt;html&gt;
+      </code>
+      <!-- Audio; You don't need to make your own audio file, but we're cool if you have one.
+        Simply name as the same tag name that you're doing and save it as an mp3
+        (e.g. anchor.mp3 for anchor). -->
+      <audio src="audio/html.mp3" controls></audio>
```
Comment (jremsik-gpsw) [inline elements/html.html]: We talked about this audio tag being removed.

Otherwise looks good.

**Comment by jremsik-gpsw** [comment]
:+1:

File: elements/html.html [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+      <code>
+        &lt;html&gt;
+      </code>
+    </h2>
+    <div class="base-words">
+      <div class="base-word-line">
+        English Base Word: <strong>Hyper Text Markup Language</strong>
+      </div>
+      <div class="base-word-line">
+        Spanish Word Equivalent: <strong></strong>
```
Comment (nwerbeckes-gpsw) [inline elements/html.html:13]: You know this one, don't you?

File: elements/html.html [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+      <p>Always remember to put this first by identifying what you are making.  You’re making an HTML document so put <code>&lt;html&gt;</code> first.</p>
+    </div>
+  </div>
+  <pre class="code-example">
+    <code>
+      <strong>&lt;html&gt;</strong>
+        &lt;head&gt;
+          &lt;title&gt;Document title&lt;/title&gt;
+        &lt;/head&gt;
+      <strong>&lt;/html&gt;</strong>
```
Comment (nwerbeckes-gpsw) [inline elements/html.html:43]: It would be nice to see an empty body tag as well.

**Comment by nwerbeckes-gpsw** [comment]
:+1: Once my two comments are addressed.

**Pull request merged**